<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>comboboxesClass</class>
 <widget class="QDialog" name="comboboxesClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>comboboxes</string>
  </property>
  <property name="accessibleName" >
   <string>comboboxesClass</string>
  </property>
  <widget class="MultiComboBox" name="comboBox_2" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>530</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking" >
    <bool>false</bool>
   </property>
   <property name="accessibleName" >
    <string>comboBox_2</string>
   </property>
   <property name="autoCompletionCaseSensitivity" >
    <enum>Qt::CaseSensitive</enum>
   </property>
  </widget>
  <widget class="MultiComboBox" name="comboBox_3" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>530</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking" >
    <bool>false</bool>
   </property>
   <property name="accessibleName" >
    <string>comboBox_3</string>
   </property>
   <property name="autoCompletionCaseSensitivity" >
    <enum>Qt::CaseSensitive</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_qt" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accessibleName" >
    <string>comboBox_qt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btTest" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="accessibleName" >
    <string>btTest</string>
   </property>
   <property name="text" >
    <string>Test</string>
   </property>
  </widget>
  <widget class="BasarTableWidget" name="tableWidget_basar" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="accessibleName" >
    <string>tableWidget_basar</string>
   </property>
  </widget>
  <widget class="MultiComboBox" name="comboBox_multi" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accessibleName" >
    <string>comboBox_multi</string>
   </property>
   <property name="editable" >
    <bool>false</bool>
   </property>
   <property name="modelColumn" >
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>MultiComboBox</class>
   <extends>QComboBox</extends>
   <header>libbasarqtwidget_multicombobox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
